p8e-api/src/main/kotlin/io/provenance/engine/service/TransactionQueryService.kt

```diff
@@ -0,0 +1,27 @@
+package io.provenance.engine.service
+
+import io.provenance.engine.domain.*
+import org.springframework.stereotype.Component
+
+@Component
+class TransactionQueryService(val rpcClient: RPCClient) {
```

Comment:
We can probably get a small unit test around this class? :badpokerface

Reply:
haha, I do have a test in the works for the EventStream itself, just haven't pushed it yet... I guess I could test that if the rpc sends us what it is supposed to we will pass it along 😉

Reply:
Took me a while to figure out a half reasonable way to mock events getting sent through the WebSocket, the documentation of Scarlet is no help, I ended up basically just mocking the RxKotlin Flowable responses from the Scarlet service and that seems to work pretty well... and the coverage is at least going up over all for this PR

![image](https://user-images.githubusercontent.com/67606767/111707975-a25d3500-880a-11eb-82a4-29d0f9ef2fbb.png)